### ISSUE TITLE
Proposal: remove lowercase and plural options

### ISSUE THREAD

**Issue by Zertz** [florianholzapfel/express-restify-mongoose#234]
These options can cause confusion in cases where the endpoint won't actually match what has been set in the name property.

```js
restify.serve(router, model, {
  name: 'Model',
  lowercase: true,
  plural: true
})
```

Here, the endpoint will actually be `/api/v1/models` and not `/api/v1/Model`

Thoughts?

**Comment by Quixomatic** [comment]
I believe plural should just bet set to false by default and when plural is set to true plural should override the name property whether it is being used or not.

**Comment by florianholzapfel** [comment]
After having a look at the sample, I think it is correct that the endpoint will be ```/api/v1/models```.

Removing the ```plural``` and ```lowercase``` options are breaking changes. Existing users could easily migrate using the ```name``` option. But we should have a warn message.

In general, I think the ```name``` option is sufficient as it provides enough flexibility and does not make any assumptions on the endpoint naming.

**Comment by Zertz** [comment]
Posting this on Twitter, let's try to get someone new involved here :)

The options should be removed from [defaults](https://github.com/florianholzapfel/express-restify-mongoose/blob/master/src/express-restify-mongoose.js#L8) along with the [code](https://github.com/florianholzapfel/express-restify-mongoose/blob/master/src/express-restify-mongoose.js#L117) using them. There's also a [test case](https://github.com/florianholzapfel/express-restify-mongoose/blob/4f40b54e338718c243d438fbb00722a94fe7de11/test/integration/options.js#L46) that will need to be updated.

You can install MongoDB to run tests locally or send the PR and tests will be run automatically.

**Comment by Zertz** [comment]
Closed by #238 and #239
